src/main/java/sirius/biz/codelists/LookupTable.java

```diff
@@ -751,18 +751,31 @@ public static Map<String, ObjectNode> parseMap(ObjectNode root, JsonPointer path
     }
 
     /**
-     * Suggests several entries for the given search term using the currently active language.
+     * Suggests several entries for the given search term aside from deprecated entries using the currently
+     * active language.
      *
      * @param searchTerm the term used to filter the suggestions
      * @return a stream of suggestions for the given term. Note that most probably {@link Stream#limit(long)} should
      * be used on the result as this might yield quite a bunch of suggestions in order to optimize internal queries.
      */
     public Stream<LookupTableEntry> suggest(@Nullable String searchTerm) {
-        return suggest(searchTerm, NLS.getCurrentLanguage());
+        return suggest(searchTerm, NLS.getCurrentLanguage(), false);
     }
 
     /**
-     * Suggests several entries for the given search term using the given language.
+     * Suggests several entries for the given search term using the currently active language.
+     *
+     * @param searchTerm               the term used to filter the suggestions
+     * @param considerDeprecatedValues whether deprecated values should be suggested as well
+     * @return a stream of suggestions for the given term. Note that most probably {@link Stream#limit(long)} should
+     * be used on the result as this might yield quite a bunch of suggestions in order to optimize internal queries.
+     */
+    public Stream<LookupTableEntry> suggest(@Nullable String searchTerm, boolean considerDeprecatedValues) {
+        return suggest(searchTerm, NLS.getCurrentLanguage(), considerDeprecatedValues);
+    }
```

Comment:
**Optional:** This overload feels a bit pointless in my opinion, as `NLS.getCurrentLanguage()` is available to be called from everywhere (even in templates now as we can call static functions). Would prefer to keep the number of overloads lower in this case as there already are quite a bunch of them, which can get confusing when trying to decide which one is the right one to call.